include/json/neu_json_mqtt.h

```diff
@@ -43,10 +43,18 @@ typedef struct {
     char *command;
 } neu_json_mqtt_t;
 
+typedef struct {
+    char *   config_name;
+    char *   node_name;
+    uint32_t node_id;
+    uint64_t timestamp;
+} neu_json_mqtt_periodic_t;
+
```

Comment:
put it to read/write json, eKuiper will also use it

Reply:
Done.